tools/api-markdown-documenter/src/api-item-transforms/TransformApiItem.ts

```diff
@@ -130,93 +131,123 @@ export function apiItemToSections(
 		return [];
 	}
 
-	const logger = config.logger;
+	const { logger, transformations } = config;
+
+	const transformChildren = (childItem: ApiItem): SectionNode[] =>
+		apiItemToSections(childItem, config);
 
-	logger.verbose(`Rendering section for ${apiItem.displayName}...`);
+	logger.verbose(`Generating documentation section for ${apiItem.displayName}...`);
 
 	let sections: SectionNode[];
 	switch (apiItem.kind) {
```

Comment:
Note: the exhaustive switch statement is still required since each transformation is strongly typed over its `ApiItem` implementation. Hoping to simplify this in the future.